internal/server/authn/method/github/server.go

```diff
@@ -240,9 +259,9 @@ func api(ctx context.Context, token *oauth2.Token, endpoint endpoint, v any) err
 	return json.NewDecoder(resp.Body).Decode(v)
 }
 
-func getUserOrgs(ctx context.Context, token *oauth2.Token) (map[string]bool, error) {
+func getUserOrgs(apiURL string, ctx context.Context, token *oauth2.Token) (map[string]bool, error) {
```

Comment:
```suggestion
func getUserOrgs(ctx context.Context, apiURL string, token *oauth2.Token) (map[string]bool, error) {
```

nit: its customary for `context.Context` to be the first argument